tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/builtinconstraints/FuturePastConstraintsTest.java

```diff
@@ -422,6 +482,28 @@ public PastOrPresentDummyEntity(ZonedDateTime dateTime) {
 		}
 	}
 
+	public class PastRelativePartialDummyEntity {
```

Comment:
Not really, it started failing as the tests were coming from another package. So I decided to fix it by bringing them back here instead of relying on unrelated entities.